src/servers/src/http.rs

```diff
@@ -912,6 +920,25 @@ impl HttpServer {
             .with_state(log_state)
     }
 
+    fn route_pipelines<S>(log_state: LogState) -> Router<S> {
+        Router::new()
+            .route("/pipelines/ingest", routing::post(event::log_ingester))
```

Comment:
We need to ensure the pipeline name can't be `ingest`.

But I think changing the `ingest` to `_ingest` would be better.

Reply:
Same as `dryrun` below.

Reply:
Changed `dryrun` to `_dryrun`. And `pipelines/ingest` to `ingest` according to the discussion with @sunng87